.github/workflows/daily-nightly-jobs.yml

```diff
@@ -1,7 +1,10 @@
 name: Daily nightly jobs
 on:
-  schedule:
-    - cron: "0 0 * * *" # every day at midnight
+#  schedule: commented for testing
+#    - cron: "0 0 * * *" # every day at midnight
+  pull_request:
```

Comment:
After we confirm reef is passing and this is reverted, please also squash to a single commit.